Frends.Salesforce.ExecuteQuery/CHANGELOG.md

```diff
@@ -3,3 +3,7 @@
 ## [1.0.0] - 2022-03-07
 ### Added
 - Initial implementation of Frends.Salesforce.ExecuteQuery
+
+## [1.0.1] - 2022-05-27

```

Comment:
New version should be above the last version. This way the version are in descending order.